misc/inmanta-agent.service

```diff
@@ -6,6 +6,8 @@ After=network.target
 Type=simple
 EnvironmentFile=/etc/sysconfig/inmanta-agent
 ExecStart=/usr/bin/inmanta --log-file /var/log/inmanta/agent.log --log-file-level 2 --timed-logs agent
+StandardOutput=append:/var/log/inmanta/agent.out
```

Comment:
Did you test this? I did a quick test on centos7 and I get: `Failed to parse output specifier, ignoring: append:/var/log/inmanta/agent.out` when I run `systemctl status inmanta-agent.service`. These options are also not present in the `systemd.service` man page. I assume this is a pretty new feature of systemd and not yet included in centos7.

Reply:
It works on my system. I'll look into it, you're probably right.

Reply:
It's in `man systemd.exec` for me. But it appears it is indeed too new to be present in Centos 7. It should be there in 8. I'll use another method to redirect.